src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java

```diff
@@ -552,6 +587,25 @@ public void makeHttpRequestsShouldReturnErrorIfNoValidSizes() {
         assertThat(result.getValue()).hasSize(1);
     }
 
+    @Test
+    public void makeHttpRequestsShouldReturnerrorIfSizeIdsNotFound() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder()
+                        .banner(Banner.builder().build())
+                        .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpRubicon.builder()
+                                .sizes(singletonList(3)).build())))
+                        .build()))
+                .build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = rubiconBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).hasSize(1);
+        assertThat(result.getErrors()).containsOnly(BidderError.badInput("Bad request.imp[].ext.rubicon.sizes"));
```

Comment:
This can be fluent calls:
```
assertThat(result.getErrors()).hasSize(1)
    .containsOnly(BidderError.badInput("Bad request.imp[].ext.rubicon.sizes"));
```